File: android/app/src/main/java/com/post/notes/widget/utils/note_text_size/NoteTextSize.java
package com.post.notes.widget.utils.note_text_size;


public class NoteTextSize {
    private static final String TINY = "tiny";
    private static final String SMALL = "small";
    private static final String NORMAL = "normal";
    private static final String LARGE = "large";
    private static final String HUGE = "huge";

    public static int toFontSize(String type) {
        switch (type) {
            case (TINY): {
                return 14;
            }

            case (SMALL): {
                return 16;
            }

            case (NORMAL): {
                return 20;
            }

            case (LARGE): {
                return 24;
            }

            case (HUGE): {
                return 28;
            }

            default: {
                return 20;
            }
        }
    }
}

File: src/data/common/note-text-size/NoteTextSize.js
const NoteTextSize = {
  TINY: 'tiny',
  SMALL: 'small',
  NORMAL: 'normal',
  LARGE: 'large',
  HUGE: 'huge',
  toFontSize: type => {
    switch (type) {
      case NoteTextSize.TINY: {
        return 14;
      }

      case NoteTextSize.SMALL: {
        return 16;
      }

      case NoteTextSize.NORMAL: {
        return 20;
      }

      case NoteTextSize.LARGE: {
        return 24;
      }

      case NoteTextSize.HUGE: {
        return 28;
      }

      default: {
        return 20;
      }
    }
  },
};

export default NoteTextSize;

// const NoteTextSize = {
//   TINY: 'tiny',
//   SMALL: 'small',
//   NORMAL: 'normal',
//   LARGE: 'large',
//   HUGE: 'huge',
//   toFontSize: type => {
//     switch (type) {
//       case NoteTextSize.TINY: {
//         return 12;
//       }
//
//       case NoteTextSize.SMALL: {
//         return 16;
//       }
//
//       case NoteTextSize.NORMAL: {
//         return 18;
//       }
//
//       case NoteTextSize.LARGE: {
//         return 23;
//       }
//
//       case NoteTextSize.HUGE: {
//         return 26;
//       }
//
//       default: {
//         return 16;
//       }
//     }
//   },
// };
//
// export default NoteTextSize;
